src/gnc/orbit_controller.cpp

```diff
@@ -39,26 +39,26 @@
 #include <lin/math.hpp>
 #include <lin/queries.hpp>
 
-#include <cstdint>
 #include <cmath>
+#include <cstdint>
 #include <math.h>
 
 namespace gnc {
 
 // Initialize constants
-static constexpr double mass = 3.7;             // Mass of spacecraft            (kg)
-static constexpr double J_min = 0;              // Minimum impulse               (N s)
-static constexpr double J_max = 2.5e-2;         // Maximum impulse               (N s)
-static constexpr double max_dv = J_max / mass;  // Max velocity change           (m/s)
-static constexpr double min_dv = J_min / mass;  // Min velocity change           (m/s)
+static constexpr double mass = 3.7;     // Mass of spacecraft            (kg)
+static constexpr double J_min = 0;      // Minimum impulse               (N s)
+static constexpr double J_max = 2.5e-2; // Maximum impulse               (N s)
+static constexpr double max_dv = J_max / mass; // Max velocity change (m/s)
+static constexpr double min_dv = J_min / mass; // Min velocity change (m/s)
```

Comment:
What are the functional changes in this file? If it's not that many we should revert this file to what's on master, just apply those changes, and leave the diff as minimal as possible so we don't obliterate the git blame here.